client/src/views/Home.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <v-container fluid>
+        <v-row>
+            <v-col cols="3">
```

Comment:
I'm not sure if RWD matters here, but it I think it should be `cols="12" md="3"`  and below `cols="12" md="9"`